CHANGELOG.md

```diff
@@ -212,10 +297,78 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 - Fix chaos-kernel build, mark bcc version [#2693](https://github.com/chaos-mesh/chaos-mesh/pull/2693)
 - Fix wrong field name of PhysicalMachineChaos on Chaos Dashboard [#2724](https://github.com/chaos-mesh/chaos-mesh/pull/2724)
 - Fix field descriptions of GCPChaos [#2791](https://github.com/chaos-mesh/chaos-mesh/pull/2791)
-- Fix `real_gettimeofday` on arm64 [#2849](https://github.com/chaos-mesh/chaos-mesh/pull/2849)
-- Fix Github Action `upload-image` [#2935](https://github.com/chaos-mesh/chaos-mesh/pull/2935)
-- Fix JVMChaos to handle the situation that the container which holds the JVM rules has been deleted [#2981](https://github.com/chaos-mesh/chaos-mesh/pull/2981)
-- Fix typo in comments for Chaos API [#3109](https://github.com/chaos-mesh/chaos-mesh/pull/3109)
```

Comment:
Why delete these?

Reply:
This bug-fixes already carried with `2.1.5`, which is earlier than the `2.2.0`, so they should be deleted from `Unreleased` section before we release 2.2.0. So I remove them.

Reply:
@STRRL We should keep these original changes (if they have cherry-picks into old versions).

Also, if we change these, they won't be consistent with the release notes.

Reply:
Yes. You're right, keep tracking the cherrypicked PR and making it keep consistent would cost so much.....

I would append them back.

BTW, maybe we should not manually CHANGELOG.md anymore, instead of using PR's title...

Reply:
> BTW, maybe we should not manually CHANGELOG.md anymore, instead of using PR's title...

🤔 You mean some lines haven't been consistent with the original PR titles? I think it's fine as long as you can describe what PR does because this is somewhat difficult to constrain everyone to do so.

Reply:
Yes. It could be hard. GitHub could actually generate the CHANGELOG based on PRs, and here are some practices called "Semantic PRs". 👀

Reply:
But auto-generation PRs from GitHub will be generated only before we prepare to release a new version. So it still needs to constrain the format in the CHANGELOG because it will be updated earlier.